hs_core/management/commands/fix_buckets.py

```diff
@@ -6,14 +6,14 @@
 
 
 class Command(BaseCommand):
-    help = "Analyze and remove erroneous bucket names"
+    help = "Analyze erroneous bucket names"
```

Comment:
This management command can be deleted. It was used during development when I found the <3 character constraint.